packages/kuma-gui/src/app/application/index.ts

```diff
@@ -260,9 +258,6 @@ export const services = (app: Record<string, Token>): ServiceDefinition[] => {
         app.sources,
       ],
     }],
-
-    ...me(app),
-    ...x(app),
     ...kuma(app),
```

Comment:
I moved `me` and `x` to be part of `service-mesh` for now. They shouldn't be part of (a generic) `application` module. `kuma` ideally shouldn't be here either but I'd need to do more re-plumbing to move that, so I left that for now.